zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -7085,6 +7196,44 @@ components:
         email:
           description: The email of the user.
           type: "string"
+    TenantMemberResult:
+      type: object
+      properties:
+        memberId:
+          description: The ID of the member.
+          type: string
+    TenantMemberSearchResult:
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryResponse"
+      properties:
+        items:
+          description: The matching clients.
```

```suggestion
          description: The matching members.
```

Found this while copy-pasting your code 😄

---

Fixed in ff294f18723a04d32252424bbd7b3b3ce82d3e47